### ISSUE TITLE
Command failed: npm install cgb-scripts

### ISSUE THREAD

**Issue by tcmulder** [ahmadawais/create-guten-block#136]
I'm having issues running the initial `npx create-guten-block my-block` command.

I've tried the following (mostly from [ideas in this thread](https://github.com/remy/nodemon/issues/1124)):

1. Reinstalling Node (using the download from Node's site).
2. Manually deleting anything Node, npm, and nvm off my computer and _then_ reinstalling them.
3. The same again with a restart before the reinstallation.
4. Switching from Node version 10.15.0 (current stable) to 8.1.0, 8.2.1, and 9.1.0.
5. Per the only somewhat related [issue for this project](https://github.com/ahmadawais/create-guten-block/issues/99), I updated npm from what comes with Node's download to 6.4.1.

After each of these I ran `npx create-guten-block my-block` again and get the same error. The full error output is as follows:
```
✔ 1. Creating the plugin directory called →  my-block
⠹ 2. Installing npm packages.../usr/local/lib/node_modules/create-guten-block/index.js:60
	throw err;
	^

Error: Command failed: npm install cgb-scripts --save --save-exact --slient
module.js:544
    throw err;
    ^
Error: Cannot find module 'internal/util/types'
    at Function.Module._resolveFilename (module.js:542:15)
    at Function.Module._load (module.js:472:25)
    at Module.require (module.js:585:17)
    at require (internal/module.js:11:18)
    at evalmachine.<anonymous>:31:26
    at Object.<anonymous> (wp-content/node_modules/npm/node_modules/graceful-fs/fs.js:11:1)
    at Module._compile (module.js:641:30)
    at Object.Module._extensions..js (module.js:652:10)
    at Module.load (module.js:560:32)
    at tryModuleLoad (module.js:503:12)


    at makeError (/usr/local/lib/node_modules/create-guten-block/node_modules/execa/index.js:169:9)
    at Promise.all.then.arr (/usr/local/lib/node_modules/create-guten-block/node_modules/execa/index.js:274:16)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

Note that the Node <= 9 tests changed `Error: Cannot find module 'internal/util/types'` to `Error: Cannot find module 'internal/fs'`, but the rest looked similar.

I'm on Mac OS 10.14.2, currently running node 10.15.0 and npm 6.4.1. 

Any ideas as to what could cause this issue?

**Comment by ahmadawais** [comment]
Kindly, file a bug report with answers to all the questions there.

**Comment by sasigit7** [comment]
@tcmulder - have u managed to fix the issue? I have got the same exact problem. I have tried almost similar steps like you to fix it but to no avail. Plz advise. Thanks.

` Creating a WP Gutenberg Block plugin called:  test-block 
 
 In the directory: /Applications/XAMPP/xamppfiles/htdocs/gutenberg/wp-content/plugins/test-block
 This might take a couple of minutes.

✔ 1. Creating the plugin directory called →  test-block 
⠙ 2. Installing npm packages.../usr/local/lib/node_modules/create-guten-block/index.js:60
        throw err;
        ^

Error: Command failed: npm install cgb-scripts --save --save-exact --silent


    at makeError (/usr/local/lib/node_modules/create-guten-block/node_modules/execa/index.js:169:9)
    at /usr/local/lib/node_modules/create-guten-block/node_modules/execa/index.js:274:16
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async /usr/local/lib/node_modules/create-guten-block/app/npmInstallScripts.js:48:3 {
  code: 1,
  stdout: '',
  stderr: '',
  failed: true,
  signal: null,
  cmd: 'npm install cgb-scripts --save --save-exact --silent',
  timedOut: false,
  killed: false
}`
